packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -0,0 +1,150 @@
+import { Component, h, JSX, Listen, Element, Prop, State } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import {
+  validateFormIds,
+  getSlottedInput
+} from '../../../stable/gux-form-field/gux-form-field.service';
+import { GuxSelectorCardVariant } from './gux-selector-card.types';
+import { calculateInputDisabledState } from '@utils/dom/calculate-input-disabled-state';
+import { onInputDisabledStateChange } from '@utils/dom/on-input-disabled-state-change';
+import { preventBrowserValidationStyling } from '@utils/dom/prevent-browser-validation-styling';
+import { logWarn } from '@utils/error/log-error';
+import { setSlotAriaDescribedby } from '../../../stable/gux-form-field/gux-form-field.service';
+
+@Component({
+  styleUrl: 'gux-selector-card.scss',
+  tag: 'gux-selector-card-beta',
+  shadow: true
+})
+export class GuxSelectorCard {
+  private input: HTMLInputElement;
+  private disabledObserver: MutationObserver;
+
+  @Element()
+  private root: HTMLElement;
+
+  //TODO: COMUI-2784 Descriptive variant styling
+  @Prop()
+  variant: GuxSelectorCardVariant = 'simple';
+
+  @State()
+  private disabled: boolean;
+
+  @State()
+  private checked: boolean;
+
+  componentWillLoad(): void {
+    this.setInput();
+    trackComponent(this.root, { variant: this.variant });
+    this.validateInputSlot(this.root.querySelector('[slot=input]'));
+    this.validateLabelSlot(this.root.querySelector('[slot=label]'));
+  }
+
+  disconnectedCallback(): void {
+    if (this.disabledObserver) {
+      this.disabledObserver.disconnect();
+    }
+  }
+
+  @Listen('click')
+  onClick(e: MouseEvent): void {
+    e.stopPropagation();
+    this.setChecked();
+  }
+
+  private setInput(): void {
+    this.input = getSlottedInput(this.root, 'input[slot="input"]');
+    this.checked = this.input.checked;
+    preventBrowserValidationStyling(this.input);
+
+    setSlotAriaDescribedby(this.root, this.input, 'content');
+
+    this.disabled = calculateInputDisabledState(this.input);
+
+    this.disabledObserver = onInputDisabledStateChange(
+      this.input,
+      (disabled: boolean) => {
+        this.disabled = disabled;
+      }
+    );
+
+    validateFormIds(this.root, this.input);
+  }
+
+  private validateInputSlot(slottedElement: HTMLInputElement): void {
```

Comment:
This function can be removed if you make the changes to `setInput` in my other comment.